CHANGELOG.md

```diff
@@ -3,6 +3,8 @@
 
 # 0.15.2 - 19 Jul, 2018
 
+- Introduced support for specifying multiple directories for the Migrator
```

Comment:
This actually seems to be in wrong position in changelog and needs link to PR number

Reply:
Right, sorry, will move to unreleased part.